apps/examples/src/examples/persistence/PersistenceExample.tsx

```diff
@@ -50,26 +50,22 @@ export default function PersistenceExample() {
 	// [4]
 	if (loadingState.status === 'loading') {
 		return (
-			<div className="tldraw__editor">
+			<div style={{ padding: 32 }}>
 				<h2>Loading...</h2>
 			</div>
 		)
 	}
 
 	if (loadingState.status === 'error') {
 		return (
-			<div className="tldraw__editor">
+			<div style={{ padding: 32 }}>
 				<h2>Error!</h2>
 				<p>{loadingState.error}</p>
 			</div>
 		)
 	}
```

Comment:
scout cleaning: These loading and error states looked dodgy as they were, so I replaced the classname with some super simple padding.